src/common.ts

```diff
@@ -102,17 +103,22 @@ export const createAtoms = <
       const resultAtom = get(baseStatusAtom)
       return get(resultAtom)
     },
-    (get, set, action: Action) => {
+    (get, set, action: Action | 'refresh') => {
       const observer = get(observerAtom)
       const refresh = () => {
         const queryClient = getQueryClient(get)
         const observerCache = get(observerCacheAtom)
         observerCache.delete(queryClient)
         set(refreshAtom, (c) => c + 1)
       }
+      if (action === 'refresh') {
+        refresh()
+        return observer.getCurrentResult() as unknown as ActionResult
+      }
       return handleAction(action, observer, refresh)
     }
   )
+  statusAtom.onMount = (setAtom) => () => setAtom('refresh')
```

Comment:
Hmm, this is empty `onMount` and `onUnmount` only. (It's basically violating rules if it were useEffect).

Reply:
What's your suggestion on that then?

Reply:
I don't have a suggestion for this one. It just feels like the entire implementation isn't ideal. It's somewhat convincing because `jotai-urql` did a full rewrite. Anyway, it's not about this PR. This PR should be good to go.

Reply:
I get it, at some point, we can follow the urql one step by step. That'd be a good start I guess. 
Thanks.